docs/pages/guides/mesheryctl/index.md

```diff
@@ -12,8 +12,11 @@ list: exclude
 Guides to using Meshery's various features and components.
 
 {% capture tag %}
-
-<li><a href="{{ site.baseurl }}/guides/upgrade#upgrading-meshery-cli">Upgrading mesheryctl</a></li>
+<li><a href="{{ site.baseurl }}/guides/mesheryctl/working-with-mesheryctl">Using mesheryctl</a></li>
```

Comment:
@Vidyaa123, thanks for taking on this issue. Quality work.

Let's make this dynamic and less fragile. Will you please consider a query for `type: Guides` of `category:mesheryctl`?

Reply:
@leecalcote, I have added dynamic links to the mesheryctl guides except upgrading which links to a header. Also, the order of the mesheryctl guides is different to the side navigation. Should I add an order to the guides and sort them based on that.

Reply:
@Vidyaa123 sorting alphabetically will do the trick.